openmls/src/group/core_group/new_from_external_init.rs

```diff
@@ -108,6 +108,8 @@ impl CoreGroup {
             group_epoch_secrets,
             message_secrets_store,
             own_leaf_index,
+            // TODO
```

Comment:
TODO what? Either add an issue for the todo so everyone knows what's needed, or remove.